src/index.html

```diff
@@ -16,17 +16,31 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="page">
+    <div class="search__box">
+      <form
+        class="search-bar search-bar__big"
+        data-qa="big"
+      >
```

Comment:
The class names you've used here don't seem to represent the meaning of the content. Class names should be descriptive and reflect the meaning of the content, not the styles or tag names. For example, instead of `search-bar__big` you might use `search-bar--large`.